04-loop.md

```diff
@@ -39,9 +39,10 @@ This wouldn't back up our files, instead we get an error
 mv: target `original-*.dat' is not a directory
 ~~~
 
-This is because there are no files matching the wildcard `original-*.dat`.
-In this case,
-Bash passes the unexpanded wildcard as a parameter to the `mv` command.
+This is a problem arises when mv receives more than two inputs. When this happens, it 
```

Comment:
Wrap "mv" with backtick quotes:

~~~
This is a problem arises when `mv` receives more than two inputs. When this happens, it
~~~